In [1]:
name = "Ray"
hourly_wage = 17.50
hours = 10
weekly_pay = hourly_wage * hours
print(f"{name} earns {weekly_pay} per week")

Ray earns 175.0 per week


In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]
warm_days = 0
for t in temps:
    if t > 65:
        warm_days = warm_days + 1
print(warm_days)

4


In [3]:
def to_celsius(f):
    return (f - 32) * 5/9

result = to_celsius(temps[0])
print(round(result, 1))

16.1


In [4]:
movie = {
    "title": "The Bee Movie",
    "year": 2007,
    "rating": 9  
}

for key, value in movie.items(): 
    print(key, value)

title The Bee Movie
year 2007
rating 9


In [5]:
import pandas as pd

In [6]:
df = pd.read_csv("chickwts.csv", index_col=0)
df.head()

,weight,feed
1,179,horsebean
2,160,horsebean
3,136,horsebean
4,227,horsebean
5,217,horsebean


In [7]:
df.shape

(71, 2)

In [8]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


The mean is 261g and the median is 258, so they are close meaning the weights are fairly symmetric overall. The spread is large(the std is 78g with the range going from 108g to 423g). This is probably because feed is not grouped together, so the differences between the different feed types are contributing to the spread.